platform/viewer/src/googleCloud/DicomStorePickerModal.js

```diff
@@ -12,12 +12,17 @@ class DicomStorePickerModal extends Component {
     setServers: PropTypes.func.isRequired,
     isOpen: PropTypes.bool.isRequired,
     onClose: PropTypes.func,
+    server: PropTypes.object,
   };
 
   static defaultProps = {
     isOpen: false,
   };
 
+  setServer = () => {
+    if (this.props.server) this.props.setServers(this.props.server);
+  };
```

Comment:
We can probably continue to call `setServers`, but also call `this.props.setServers` in `mounted` hook if `this.props.server` has a value?

Reply:
Yes, that's true. Added a mounted hook with commit https://github.com/OHIF/Viewers/pull/790/commits/0d752399dd608cc891bf26b48b8c4398f9fa596a